pgtap/withPoints/withPointsDD/edge_cases/issue_979.pg

```diff
@@ -115,46 +116,31 @@ IF min_version('3.6.0') THEN
 ---
 --- DIRECTED GRAPH
 ---
--------- both driving sides
```

Comment:
This test works for on 3.5.0
Please put it back and handle it in an if then else

Reply:
Actually, this test works for 3.6.0. I didn't remove the tests work on 3.5.0, they still [there](https://github.com/squarege/GSoC-pgRouting/blob/9a1a1a4967fa1002ad4f183804325a37921cd94f/pgtap/withPoints/withPointsDD/edge_cases/issue_979.pg#L170-L262).
And I added a `TODO remove` comment before the tests work on old version.